pythoncalculator/substract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
It looks as though this line should actually be
```suggestion
    return x - y
```